app/utils/fetchWithTokenUtil.ts

```
@@ -0,0 +1,31 @@
+    ...options,
+    headers,
+  });
+
+  if (!resp.ok) {
+    throw new Error(`API 요청 실패: ${resp.statusText}`);
+  }
+
+  try {
+    const contentType = resp.headers.get('content-type');
+    if (contentType && contentType.includes('application/json')) {
+      return resp.json();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)